packages/plugin-ext/src/plugin/known-commands.ts

```diff
@@ -56,207 +57,226 @@ export namespace KnownCommands {
-    mappings['jumpToPrevSnippetPlaceholder'] = ['jumpToPrevSnippetPlaceholder', MONACO_CONVERSION_IDENTITY];
-    mappings['leaveEditorMessage'] = ['leaveEditorMessage', MONACO_CONVERSION_IDENTITY];
-    mappings['leaveSnippet'] = ['leaveSnippet', MONACO_CONVERSION_IDENTITY];
-    mappings['closeMarkersNavigation'] = ['closeMarkersNavigation', MONACO_CONVERSION_IDENTITY];
-    mappings['goToNextReferenceFromEmbeddedEditor'] = ['goToNextReferenceFromEmbeddedEditor', MONACO_CONVERSION_IDENTITY];
-    mappings['goToPreviousReferenceFromEmbeddedEditor'] = ['goToPreviousReferenceFromEmbeddedEditor', MONACO_CONVERSION_IDENTITY];
-    mappings['closeParameterHints'] = ['closeParameterHints', MONACO_CONVERSION_IDENTITY];
-    mappings['showNextParameterHint'] = ['showNextParameterHint', MONACO_CONVERSION_IDENTITY];
-    mappings['showPrevParameterHint'] = ['showPrevParameterHint', MONACO_CONVERSION_IDENTITY];
-    mappings['acceptSelectedSuggestion'] = ['acceptSelectedSuggestion', MONACO_CONVERSION_IDENTITY];
-    mappings['acceptSelectedSuggestionOnEnter'] = ['acceptSelectedSuggestionOnEnter', MONACO_CONVERSION_IDENTITY];
-    mappings['hideSuggestWidget'] = ['hideSuggestWidget', MONACO_CONVERSION_IDENTITY];
-    mappings['insertBestCompletion'] = ['insertBestCompletion', MONACO_CONVERSION_IDENTITY];
-    mappings['insertNextSuggestion'] = ['insertNextSuggestion', MONACO_CONVERSION_IDENTITY];
-    mappings['insertPrevSuggestion'] = ['insertPrevSuggestion', MONACO_CONVERSION_IDENTITY];
-    mappings['selectNextPageSuggestion'] = ['selectNextPageSuggestion', MONACO_CONVERSION_IDENTITY];
-    mappings['selectNextSuggestion'] = ['selectNextSuggestion', MONACO_CONVERSION_IDENTITY];
-    mappings['selectPrevPageSuggestion'] = ['selectPrevPageSuggestion', MONACO_CONVERSION_IDENTITY];
-    mappings['selectPrevSuggestion'] = ['selectPrevSuggestion', MONACO_CONVERSION_IDENTITY];
-    mappings['toggleSuggestionDetails'] = ['toggleSuggestionDetails', MONACO_CONVERSION_IDENTITY];
-    mappings['toggleSuggestionFocus'] = ['toggleSuggestionFocus', MONACO_CONVERSION_IDENTITY];
-    mappings['acceptRenameInput'] = ['acceptRenameInput', MONACO_CONVERSION_IDENTITY];
-    mappings['cancelRenameInput'] = ['cancelRenameInput', MONACO_CONVERSION_IDENTITY];
-    mappings['closeAccessibilityHelp'] = ['closeAccessibilityHelp', MONACO_CONVERSION_IDENTITY];
-    mappings['history.showNext'] = ['history.showNext', MONACO_CONVERSION_IDENTITY];
-    mappings['history.showPrevious'] = ['history.showPrevious', MONACO_CONVERSION_IDENTITY];
-    mappings['closeReferenceSearch'] = ['closeReferenceSearch', MONACO_CONVERSION_IDENTITY];
-    mappings['goToNextReference'] = ['goToNextReference', MONACO_CONVERSION_IDENTITY];
-    mappings['goToPreviousReference'] = ['goToPreviousReference', MONACO_CONVERSION_IDENTITY];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const VSCODE_CONVERSION_IDENTITY = (args: any[] | undefined) => {
```

Comment:
I've got rid of that necessity to use `undefined` as second argument in case of result conversion is not needed. So readability of git diff is improved